webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -2109,9 +2109,9 @@ def update_misfit_plot(
             x="REAL",
             y="ABSDIFF",
             title=ens_name,
-            range_y=[min_diff * 0.30, max_diff * 1.05],
+            range_y=[min_diff * 0.0, max_diff * 1.05],
             color="ABSDIFF",
-            range_color=[min_diff * 0.30, max_diff * 1.00],
+            range_color=[min_diff * 0.0, max_diff * 1.00],
```

Comment:
What is the reason for this change? Should it just be `0.0` instead of `min_diff * 0.0` which is anyway 0?

Reply:
I changed this to 0 as i ran into a case where some realization results were initially hidden, but i agree this was a sloppy change. I will drop the min_diff.